frontend/src/query_builder/AccordianList.jsx

```diff
@@ -157,9 +165,28 @@ export default class AccordianList extends Component {
                                 }
                             </div>
                         : null }
+
+                        { searchable &&
+                            /* NOTE: much of this structure is here just to match strange stuff in 'List-item' below so things align properly */
+                            <div className="px1 pt1">
```

Comment:
I think this is a big enough block of code that it'd be worth factoring out into its own component (maybe a stateless one) like "ListSearchField" or similar  so we prevent copypasta and style drift.